### PR TITLE
Merge branch-AboutUs-Qiaoran with Master

### LINKED ISSUE #4: Update AboutUs.md

### PR DESCRIPTION
**Pull request by Qiaoran-M**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Project Advisor
 
-### Jane Doe
+### Meng Qiaoran
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/Qiaoran.png" width="200px">
```
Comment (Benjamin-Sim) [inline docs/AboutUs.md]: I think the image filename should be your GitHub username in all lowercase as mentioned [here](https://nus-cs2103-ay2223s1.github.io/website/schedule/week7/project.html#2-update-project-website-aboutus-readme)

File: docs/AboutUs.md [hunk 2]
```diff
@@ -19,15 +19,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/Qiaoran-M)]
+[[portfolio](team/Qiaoran.md)]
```
Comment (Benjamin-Sim) [inline docs/AboutUs.md]: Same for the PPP filename as mentioned [here](https://nus-cs2103-ay2223s1.github.io/website/schedule/week7/project.html#ppp-format-2)

**Review by Benjamin-Sim — COMMENTED** [review]
All good except for filename issues

**Review by tanhl2000 — APPROVED** [review]
Looks good

**Pull request merged**